### PR TITLE
Notify user

### PR DESCRIPTION
**Pull request by i-zor066**
This is where I'm currently at. Everything works.

### CODE REVIEW

File: BlocSpot/app/src/main/java/com/izor066/android/blocspot/ui/MainActivity.java [hunk 1]
```diff
@@ -54,10 +69,40 @@ public int getIndicatorColor(int position) {
...
+                            Geofence.GEOFENCE_TRANSITION_ENTER
+//                                    | Geofence.GEOFENCE_TRANSITION_DWELL
+                                    | Geofence.GEOFENCE_TRANSITION_EXIT).build());
+        }
+
+        mGeofenceStore = new GeofenceStore(this, mGeofences);
     }
 
+    @Override
+    protected void onStart() {
```
Comment (Electryc) [inline BlocSpot/app/src/main/java/com/izor066/android/blocspot/ui/MainActivity.java]: To be honest I don't think this was the problem with the notifications. If you don't provide additional behaviour to `onStart()` then you shouldn't override it (as it anyway the `super` is called). 

The same goes for `onStop()`.

Comment (i-zor066) [inline BlocSpot/app/src/main/java/com/izor066/android/blocspot/ui/MainActivity.java]: Yeah, it wasn't, I commented it out - no change, still works. :)

File: BlocSpot/app/src/main/java/com/izor066/android/blocspot/ui/MainActivity.java [hunk 2]
```diff
@@ -54,10 +70,37 @@ public int getIndicatorColor(int position) {
...
+                    .setTransitionTypes(
+                            Geofence.GEOFENCE_TRANSITION_ENTER
+//                                    | Geofence.GEOFENCE_TRANSITION_DWELL
+                                    | Geofence.GEOFENCE_TRANSITION_EXIT).build());
+        }
+
+        mGeofenceStore = new GeofenceStore(this, mGeofences);
     }
 
+//    @Override
```
Comment (Electryc) [inline BlocSpot/app/src/main/java/com/izor066/android/blocspot/ui/MainActivity.java]: If it still works, better delete it :tongue:

Comment (i-zor066) [inline BlocSpot/app/src/main/java/com/izor066/android/blocspot/ui/MainActivity.java]: Removed.

Comment (Electryc) [inline BlocSpot/app/src/main/java/com/izor066/android/blocspot/ui/MainActivity.java]: :+1:

**Comment by Electryc** [comment]
Looking good :+1:

**Pull request merged**